src/main/java/cuchaz/enigma/command/RefreshEnigmaMappingsCommand.java

```diff
@@ -0,0 +1,48 @@
+package cuchaz.enigma.command;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import cuchaz.enigma.ProgressListener;
+import cuchaz.enigma.translation.mapping.EntryMapping;
+import cuchaz.enigma.translation.mapping.MappingFileNameFormat;
+import cuchaz.enigma.translation.mapping.MappingSaveParameters;
+import cuchaz.enigma.translation.mapping.serde.EnigmaMappingsReader;
+import cuchaz.enigma.translation.mapping.serde.EnigmaMappingsWriter;
+import cuchaz.enigma.translation.mapping.tree.EntryTree;
+
+
+/**
+ * This is a tool to mass-migrate enigma mappings to a newer format, to avoid creating unnecessary diffs when a file is first changed.
+ */
+public class RefreshEnigmaMappingsCommand extends Command {
```

Comment:
This can already be done by running `convert-mappings enigma <directory> enigma <directory>`.

Reply:
alright :(